main.go

```diff
@@ -59,6 +72,45 @@ func main() {
 	defer machine.Close(context.Background())
 
 	runTests(machine)
+	cmd := exec.Command("python3", "plot.py")
+	err = cmd.Run()
+	if err != nil {
+		logger.Error(err)
+		return
+	}
+	uploadFiles("./savedImages/sensor_ms_pos.jpg")
+	uploadFiles("./savedImages/sensor_ms_vels.jpg")
+	uploadFiles("./savedImages/sensor_spin_degs.jpg")
+	uploadFiles("./savedImages/sensor_sv_vels.jpg")
+	uploadFiles("./savedImages/wheeled_ms_pos.jpg")
+	uploadFiles("./savedImages/wheeled_ms_vels.jpg")
+	uploadFiles("./savedImages/wheeled_spin_degs.jpg")
+	uploadFiles("./savedImages/wheeled_sv_vels.jpg")
+	
+}
+
+func uploadFiles(filename string) {
+	img, err := os.ReadFile(filename)
+	if err != nil {
+		logger.Error(err)
+		return
+	}
+	bytes := bytes.NewBuffer(img)
+	fileupload.UploadJpeg(context.Background(), bytes, partID, apikey, apikeyid, logger)
```

Comment:
Pass the date and the name of the test(motor vs base) into this function, to use as tags